src/plugins/unified_search/kibana.json

```diff
@@ -9,7 +9,7 @@
   },
   "server": true,
   "ui": true,
-  "requiredPlugins": ["dataViews", "data", "uiActions", "screenshotMode"],
+  "requiredPlugins": ["dataViews", "data", "uiActions", "screenshotMode", "usageCollection"],
```

Comment:
in the `unified_search` plugin we treat `usageCollection` as an optional plugin. In future we are are going to refactor that, but for now for consistency please move it  to `optionalPlugins`. Just to don't forget to refactor code in future!